assets/component-facets.css

```diff
@@ -272,11 +272,40 @@
   padding: 0.5rem 2rem;
 }
 
+.facets-layout-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  text-align: center;
+  padding: 2rem 2.4rem;
+}
+
+.facets-layout-grid.facets__list--vertical {
+  padding: 1rem 0;
+}
+
+.facets-layout-grid.facets-layout-grid--swatch {
+  gap: 3rem 1rem;
+}
+
+.facets-layout-grid.facets-layout-grid--thumbnail {
+  gap: 1.2rem;
+}
+
 .facets__item {
   display: flex;
   align-items: center;
 }
 
+/* Hover/focus state */
+.facets-layout-list .facets__label:hover .facet-checkbox__text,
+.facets-layout-list input:focus ~ .facet-checkbox__text {
+  text-decoration: underline;
+}
```

Comment:
(remark) Although this is fine IMO, I just wanted to point out that before, the hover state on the `drawer`/`mobile` mode wouldn't apply and underline to the filter values.  Now this new selectors does have the side-effect of adding an underline style to thoses displays as well.

I do think this is a good change thought.

Reply:
Yeah the non-mobile didn't have an underline before either. Al wanted to add it, he didn't specify if it should only be applied to desktop, but I don't think we should have different styles between the two (or else we'll continue to have spaghetti code in the future)